README.md

```diff
@@ -35,9 +35,14 @@ Requires [Go](https://golang.org/) 1.6+.
 1. Run
 
     ```bash
-    fedramp-templater <openControlsDir> <inputDoc> <outputDoc>
+    # To Fill the SSP.
+    fedramp-templater -fill <openControlsDir> <inputDoc> <outputDoc>
     # i.e.
-    fedramp-templater opencontrols/ FedRAMP-System-Security-Plan-Template-v2.1.docx FedRAMP-Masonry-Template-v2.1.docx
+    fedramp-templater -fill opencontrols/ FedRAMP-System-Security-Plan-Template-v2.1.docx FedRAMP-Masonry-Template-v2.1.docx
+    
+    # To diff the SSP with the YAML
+    fedramp-templater -diff <openControlsDir> <inputDoc>
+    fedramp-templater -diff opencontrols/ FedRAMP-System-Security-Plan-Template-v2.1.docx
```

Comment:
Since it needs to be one or the other, I would pass `fill` or `diff` as an argument. This will also be more consistent with how we do sub-commands in Masonry.